### ISSUE TITLE
Do Chinese models will come?

### ISSUE THREAD

**Issue by MrRace** [UKPLab/sentence-transformers#321]
It is a good job. 
And I know the `distiluse-base-multilingual-cased` model can support Chinese, but when to measure the semantic similarity between  Chinese sentence pairs, it seem not so suitable. Therefore, would you plan to release models for Chinese .

**Comment by googlx** [comment]
Actually it's better for you to finetune your own model based on your own data. You can try it on the dataset LCQMC.

**Comment by googlx** [comment]
Actually it's better for you to finetune a model based on your own data. You can try it on the dataset LCQMC.

**Comment by MrRace** [comment]
@googlx  OK, I will try. The label value should be string or int type? In the examples, it seems to choose int or float type. In LCQMC data, the labels are "1"(similar) and "0"(dissimilar), should I set  `int` type for them? Thanks a lot!

**Comment by nreimers** [comment]
Hi @MrRace 
Thanks for the feedback. I sadly haven't tested the model for Chinese.

Do you (or someone else) know if there is a good Chinese semantic similarity test set available similar to STS benchmark for English?

I have different Chinese models on my PC and currently I am training a multilingual model for 40 languages, including Chinese. I hope to release the model soon, but training takes quite a while for so many languages. But without suitable test set, it is hard to say how good these are for Chinese.

Also have a look at this example:
https://github.com/UKPLab/sentence-transformers/blob/master/examples/training_multilingual/make_multilingual.py
https://github.com/UKPLab/sentence-transformers/blob/master/docs/training/multilingual-models.md


This allows you to extend English sentence embeddings models to new languages. For a Chinese version, you would just need to change the target language to Chinese.

For Korean, this approach works as good as training directly on suitable Korean training data (KoreanNLI and KoreanSTSbenchmark).

Best
Nils Reimers

**Comment by MrRace** [comment]
Thanks for you reply @nreimers . As I know there are some Chinese STS below:
1. LCQMC (A Large-scale Chinese Question Matching Corpus)  http://icrc.hitsz.edu.cn/Article/show/171.html
train set：238766 pairs，dev set：8802 pairs，test set：12500 pairs
2. BQ Corpus(A Large-scale Domain-specific Chinese Corpus For Sentence Semantic Equivalence Identification) http://icrc.hitsz.edu.cn/Article/show/175.html

**Comment by ares5221** [comment]
same question

**Comment by nreimers** [comment]
Hi @ares5221 @MrRace 

I just released this model:
xlm-r-40langs-bert-base-nli-stsb-mean-tokens

It was trained on 40 languages, including Chinese. The teacher model was bert-base-nli-stsb-mean-tokens.

I will update the documentation soon and add more models. Please let me know if this model works better than DistilUSE.

Best
Nils

**Comment by ares5221** [comment]
> Hi @ares5221 @MrRace
> 
> I just released this model:
> xlm-r-40langs-bert-base-nli-stsb-mean-tokens
> 
> It was trained on 40 languages, including Chinese. The teacher model was bert-base-nli-stsb-mean-tokens.
> 
> I will update the documentation soon and add more models. Please let me know if this model works better than DistilUSE.
> 
> Best
> Nils

thks for your reply, i will check it

**Comment by leleyi** [comment]
> Actually it's better for you to finetune a model based on your own data. You can try it on the dataset LCQMC.

Do I need to perform word segmentation when using LCQMC ?

**Comment by yezhanglang** [comment]
> Hi @ares5221 @MrRace
> 
> I just released this model: xlm-r-40langs-bert-base-nli-stsb-mean-tokens
> 
> It was trained on 40 languages, including Chinese. The teacher model was bert-base-nli-stsb-mean-tokens.
> 
> I will update the documentation soon and add more models. Please let me know if this model works better than DistilUSE.
> 
> Best Nils

Hi @nreimers 
How can I get the model  xlm-r-40langs-bert-base-nli-stsb-mean-tokens ? I can't found it.

**Comment by nreimers** [comment]
@yezhanglang This models is quite bad. I recommend to use one of these models listed here:
https://www.sbert.net/docs/pretrained_models.html#multi-lingual-models

They are way better. They also work for Chinese

**Comment by ralgond** [comment]
@nreimers  I have 200K pairs  of chinese sentence, are them enough to be used for training a model?

**Comment by ares5221** [comment]
您好，您发送的邮件我已经收到。我将尽快给您回复。如紧急，请电联。This is an automatic reply, confirming that your email was received.I will reply as soon as possible.Thank you

**Comment by nreimers** [comment]
Yes
